src/App.js

```diff
@@ -179,58 +161,20 @@ const quizProgress = [
 
 export default function App() {
   const { auth, setAuth } = useAuth();
+  const { quizzes, setQuizzes } = useQuiz();
```

Comment:
I think this state and the API call would be more appropriate to be in the Quiz component so that the Quiz component is more standalone. Is there a reason you added the quizzes state and API call in App.js?

Reply:
Initially it was to make the Quiz information accessible at the top level of the application, but now that the quiz information is set to a context, that isn't really an issue.  I can definitely move the quizzes state and API call to the relevant component.  Thanks!